### PR TITLE
Homogenize creation add operation

### PR DESCRIPTION
**Pull request by letypequividelespoubelles**

### CODE REVIEW

**Review by OlivierBBB — COMMENTED** [review]
There is also the `OobOperation.java`'s `callToADD` that uses `add.callADD(arg1, arg2)` where
```java
    final EWord arg1 = EWord.of(arg1Hi, arg1Lo);
    final EWord arg2 = EWord.of(arg2Hi, arg2Lo);
```
I.e. we're feeding it `EWord` rather than `Bytes32`

**Comment by letypequividelespoubelles** [comment]
> There is also the `OobOperation.java`'s `callToADD` that uses `add.callADD(arg1, arg2)` where
> 
> ```java
>     final EWord arg1 = EWord.of(arg1Hi, arg1Lo);
>     final EWord arg2 = EWord.of(arg2Hi, arg2Lo);
> ```
> 
> I.e. we're feeding it `EWord` rather than `Bytes32`

This is the only place where we call `ADD` in fact. But when I try to cast the `EWord` in a `Bytes32` intelliJ tells me it won't do anything, as `EWord` is an implementation of `Bytes32` ...

**Review by OlivierBBB — APPROVED** [review]
LGTM

File: arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java [hunk 1]
```diff
@@ -480,8 +481,8 @@ private void callToADD(
     checkArgument(arg1Lo.bitLength() / 8 <= 16);
     checkArgument(arg2Hi.bitLength() / 8 <= 16);
     checkArgument(arg2Lo.bitLength() / 8 <= 16);
-    final EWord arg1 = EWord.of(arg1Hi, arg1Lo);
-    final EWord arg2 = EWord.of(arg2Hi, arg2Lo);
+    final Bytes32 arg1 = EWord.of(arg1Hi, arg1Lo);
+    final Bytes32 arg2 = EWord.of(arg2Hi, arg2Lo);
```
Comment (OlivierBBB) [inline arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java:485]: That is very strange ^^ but it seems to not raise an issue

**Pull request merged**